graalpy-install-package/src/main/java/com/oracle/example/graalpy/PyfigletFrame.java

```diff
@@ -61,6 +61,8 @@
 import javax.swing.event.ChangeEvent;
 import javax.swing.event.ChangeListener;
 
+import org.graalvm.nativeimage.ImageInfo;
```

Comment:
According to the diff, `ImageInfo` is not being used. Are you sure it's needed here?